emoji-copy@felipeftn/prefs.js

```diff
@@ -1,82 +1,73 @@
-const { GLib, GObject, Gio, Gtk, GdkPixbuf } = imports.gi;
-
-const Gettext = imports.gettext.domain('emoji-copy');
-const _ = Gettext.gettext;
-
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-
-function init() {
-    ExtensionUtils.initTranslations();
-}
-
-const EmojiCopySettingsWidget = new GObject.Class({
-    Name: 'EmojiCopy.Prefs.Widget',
-    GTypeName: 'EmojiCopyPrefsWidget',
+import * as Gtk from 'gi://Gtk';
```

Comment:
These lines should [probably](https://gjs.guide/extensions/overview/anatomy.html#prefs-js) be something like

```js
import Gtk from 'gi://Gtk?version=4.0';
import Adw from 'gi://Adw';
import {ExtensionPreferences, gettext as _} from 'resource:///org/gnome/Shell/Extensions/js/extensions/prefs.js';
```